### DISCUSSION TITLE
package-lock resolved field wired url

### DISCUSSION THREAD

**Discussion by cangSDARM** [verdaccio/verdaccio#2941] [Q&A]
Not sure how to describe this problem.
The format of the resolved field of the lock file should be `https://registry/package/-/package-1.1.0.tgz`, but sometimes `https://registry/package/download/package-1.1.0.tgz` appears.

![image](https://user-images.githubusercontent.com/20441896/150478166-2a48165e-47dc-4b64-8517-f95c1c62998f.png)

Also, it's not quite clear if it's a problem with verdaccio or npm or our nginx.

All in all, this created unnecessary diffs and caused a lot of trouble for our reviewers.
Does anyone know what's going on?

**Comment by juanpicado** [comment]
Verdaccio does not change anything on this matter for sure.
